unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1839,33 +1757,41 @@ handleShowDefinition outputLoc showDefinitionScope inputQuery = do
       let currentNames = NamesWithHistory.fromCurrentNames $ Branch.toNames currentBranch
       let ppe = Backend.getCurrentPrettyNames hqLength (Backend.Within currentPath') root
       pure (currentNames, ppe)
+  let pped = PPED.biasTo (mapMaybe HQ.toName inputQuery) unbiasedPPED
   Backend.DefinitionResults terms types misses <- do
     let nameSearch = NameSearch.makeNameSearch hqLength names
     Cli.runTransaction (Backend.definitionsByName codebase nameSearch includeCycles NamesWithHistory.IncludeSuffixes query)
   outputPath <- getOutputPath
-  when (not (null types && null terms)) do
```

Comment:
This is where most of the big changes happened, had to re-organize a bunch of the show definition logic since the pretty-printing logic moved out of OutputMessages.